js/mviewer.js

```diff
@@ -1919,6 +1979,28 @@ mviewer = (function () {
             _map.render();
         },
 
+        /**
+         * Public Method: highlightFeatures
+         *
+         */
+        highlightFeatures: function (features) {
+            _sourceSelectOverlay.clear();
+            if (features.length > 0) {
+                _sourceSelectOverlay.addFeatures(features);
+            }
+        },
+
+        /**
+         * Public Method: highlightSubFeature
+         *
+         */
+        highlightSubFeature: function (feature) {
+            _sourceSubSelectOverlay.clear();
+            if (feature) {
+                _sourceSubSelectOverlay.addFeature(feature);
```

Comment:
Seems to be in redundancy with `highlightFeatures` method.

I really prefer to manage style instead of 2 vector layers...

Reply:
Yes, sorry, I put my arguments for two layers above.